modin/pandas/test/dataframe/test_indexing.py

```diff
@@ -675,8 +683,7 @@ def test_reindex_like():
         columns=["temp_celsius", "windspeed"],
         index=pd.DatetimeIndex(["2014-02-12", "2014-02-13", "2014-02-15"]),
     )
-    with pytest.warns(UserWarning):
-        df2.reindex_like(df1)
+    df2.reindex_like(df1)
```

Comment:
Why removed? It's [still raising default-to-pandas warning](https://github.com/modin-project/modin/blob/689faeef9e4be7b8308703a3a57db0049a109e4f/modin/pandas/base.py#L1935-L1943). As I see from the changes above, this PR tends to not remove existed warning-catchers even if the whole file is marked to ignore such warnings.

Reply:
Done. Thanks for catching this.